vlmeval/vlm/xgen_mm.py

```diff
@@ -25,7 +25,9 @@ def __init__(self, model_path='Salesforce/xgen-mm-phi3-mini-instruct-dpo-r-v1.5'
             model_path, trust_remote_code=True, use_fast=False, legacy=False
         )
         tokenizer = model.update_special_tokens(tokenizer)
+        tokenizer.eos_token = '<|end|>'
         tokenizer.padding_side = 'left'
+        
```

Comment:
Hi, the flake8 tool has identified an error in the `file vlmeval/vlm/xgen_mm.py at line 30, where there is a blank line containing whitespace (W293)`. Please modify this accordingly. Thank you!

Reply:
Done